Demo/Application/Base/AppDelegate.swift

```diff
@@ -0,0 +1,81 @@
+import Foundation
+
+@objcMembers class AppDelegate: UIResponder, UIApplicationDelegate {
```

Comment:
Do we need `@objcMembers` here?

Reply:
We need the class name to be available to ObjC to reference [here](https://github.com/braintree/braintree_ios/blob/c393c47b99c23357cd4250a4fd81a5a9444b6996/Demo/Application/Supporting%20Files/main.m#L8). Though, we can just use `@objc` instead of `@objcMembers`

Reply:
Addressed in dc311d2bb8860d88472b04f886b40b4358ac1c75